java-frontend/src/test/files/se/debug/DebugMethodYieldsCheck.java

```diff
@@ -14,4 +25,25 @@ Object bar(boolean b) { // Compliant - we do not have method behaviors for overr
   }
 
   abstract Object qix(boolean b); // Compliant
+
+  static void plop() { // Compliant - too many steps
```

Comment:
This is going to be a "long" test to run and will add unecessary time for build. We should probably work out some solution to  this problem by not actually reaching the limit we use in production.

Reply:
Added a test with a parametrized max step.